client/src/components/reviewSuggestions/Individual/Individual.js

```diff
@@ -508,7 +513,7 @@ function RenderProfileDetails(identity) {
     // }
     return (<Row>
         <Col md={4} xs={4} sm={4} lg={4} xl={4}>
-            <img className="userimg_individual" src={identity.identity.identityImageEndpoint} onError={(e)=>{e.target.onerror = null; e.target.src='https://directory.weill.cornell.edu/api/v1/person/profile/msr2004.png?returnGenericOn404=true'}}  width="80" style={{ float: "left" }} />
+            <img className="userimg_individual" src={identity.identity.identityImageEndpoint} onError={(e) => { e.target.onerror = null; e.target.src = 'https://directory.weill.cornell.edu/api/v1/person/profile/msr2004.png?returnGenericOn404=true' }} width="80" style={{ float: "left" }} />
```

Comment:
Instead of generic image from the URL with a network call. Wouldn't be better to have a static image of the generic picture as asset inside client and refer that

Reply:
@vav4001 Let me know if this can be done.

Reply:
Will be taken care of in future PR.